doc/source/reference/random/multithreading.rst

```diff
@@ -12,31 +12,27 @@ these requirements.
 This example makes use of Python 3 :mod:`concurrent.futures` to fill an array
 using multiple threads.  Threads are long-lived so that repeated calls do not
 require any additional overheads from thread creation. The underlying
-BitGenerator is `PCG64` which is fast, has a long period and supports
```

Comment:
@bashtage thoughts on the wording here?

Reply:
Oups, yes, this isn't ideal! As I will use `default_rng` I might just delete this sentence altogether.